#! /bin/sh /usr/share/dpatch/dpatch-run
## 02_add_dtddecl_and_force_fpi_over_sysid.dpatch by Adam Di Carlo <aph@debian.org>.
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: * docbook-4.x/docbook.cat: Adds DTDDECL declarations for convenience.
## DP:   Forces FPI to override SYSTEM IDs, if both are supplied.
## DP:
## DP: <URL:http://packages.debian.org/changelogs/pool/main/d/docbook-xml/current/changelog#versionversion4.1.2-2>
## DP: <URL:http://packages.debian.org/changelogs/pool/main/d/docbook-xml/current/changelog#versionversion4.2-8>
## DP: <URL:http://packages.debian.org/changelogs/pool/main/d/docbook-xml/current/changelog#versionversion4.5-1>

@DPATCH@
diff -urNad docbook-xml-4.5~/docbook-4.1.2/docbook.cat docbook-xml-4.5/docbook-4.1.2/docbook.cat
--- docbook-xml-4.5~/docbook-4.1.2/docbook.cat	2007-03-25 21:45:05.000000000 +0200
+++ docbook-xml-4.5/docbook-4.1.2/docbook.cat	2007-03-25 21:47:30.000000000 +0200
@@ -16,10 +16,14 @@
      on supplying and using catalog data.
   --
 
+  -- public identifiers override system identifiers, if both are supplied   --
+OVERRIDE YES
+
   -- ...................................................................... --
   -- DocBook driver file .................................................. --
 
 PUBLIC "-//OASIS//DTD DocBook XML V4.1.2//EN" "docbookx.dtd"
+DTDDECL "-//OASIS//DTD DocBook XML V4.1.2//EN" "/usr/share/xml/declaration/xml.dcl"
 
   -- ...................................................................... --
   -- DocBook modules ...................................................... --
diff -urNad docbook-xml-4.5~/docbook-4.2/docbook.cat docbook-xml-4.5/docbook-4.2/docbook.cat
--- docbook-xml-4.5~/docbook-4.2/docbook.cat	2007-03-25 21:45:05.000000000 +0200
+++ docbook-xml-4.5/docbook-4.2/docbook.cat	2007-03-25 21:48:01.000000000 +0200
@@ -16,11 +16,16 @@
      on supplying and using catalog data.
   --
 
+  -- public identifiers override system identifiers, if both are supplied   --
+OVERRIDE YES
+
   -- ...................................................................... --
   -- DocBook driver file .................................................. --
 
 PUBLIC "-//OASIS//DTD DocBook XML V4.2//EN"
        "docbookx.dtd"
+DTDDECL "-//OASIS//DTD DocBook XML V4.2//EN"
+        "/usr/share/xml/declaration/xml.dcl"
 
   -- ...................................................................... --
   -- DocBook modules ...................................................... --
diff -urNad docbook-xml-4.5~/docbook-4.3/docbook.cat docbook-xml-4.5/docbook-4.3/docbook.cat
--- docbook-xml-4.5~/docbook-4.3/docbook.cat	2007-03-25 21:45:05.000000000 +0200
+++ docbook-xml-4.5/docbook-4.3/docbook.cat	2007-03-25 21:55:39.000000000 +0200
@@ -16,11 +16,16 @@
      on supplying and using catalog data.
   --
 
+  -- public identifiers override system identifiers, if both are supplied   --
+OVERRIDE YES
+
   -- ...................................................................... --
   -- DocBook driver file .................................................. --
 
 PUBLIC "-//OASIS//DTD DocBook XML V4.3//EN"
        "docbookx.dtd"
+DTDDECL "-//OASIS//DTD DocBook XML V4.3//EN"
+        "/usr/share/xml/declaration/xml.dcl"
 
   -- ...................................................................... --
   -- DocBook modules ...................................................... --
diff -urNad docbook-xml-4.5~/docbook-4.4/docbook.cat docbook-xml-4.5/docbook-4.4/docbook.cat
--- docbook-xml-4.5~/docbook-4.4/docbook.cat	2007-03-25 21:45:05.000000000 +0200
+++ docbook-xml-4.5/docbook-4.4/docbook.cat	2007-03-25 21:48:30.000000000 +0200
@@ -16,11 +16,16 @@
      on supplying and using catalog data.
   --
 
+  -- public identifiers override system identifiers, if both are supplied   --
+OVERRIDE YES
+
   -- ...................................................................... --
   -- DocBook driver file .................................................. --
 
 PUBLIC "-//OASIS//DTD DocBook XML V4.4//EN"
        "docbookx.dtd"
+DTDDECL "-//OASIS//DTD DocBook XML V4.4//EN"
+        "/usr/share/xml/declaration/xml.dcl"
 
   -- ...................................................................... --
   -- DocBook modules ...................................................... --
diff -urNad docbook-xml-4.5~/docbook-4.5/docbook.cat docbook-xml-4.5/docbook-4.5/docbook.cat
--- docbook-xml-4.5~/docbook-4.5/docbook.cat	2007-03-25 21:45:05.000000000 +0200
+++ docbook-xml-4.5/docbook-4.5/docbook.cat	2007-03-25 21:55:23.000000000 +0200
@@ -16,11 +16,16 @@
      on supplying and using catalog data.
   --
 
+  -- public identifiers override system identifiers, if both are supplied   --
+OVERRIDE YES
+
   -- ...................................................................... --
   -- DocBook driver file .................................................. --
 
 PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN"
        "docbookx.dtd"
+DTDDECL "-//OASIS//DTD DocBook XML V4.5//EN"
+        "/usr/share/xml/declaration/xml.dcl"
 
   -- ...................................................................... --
   -- DocBook modules ...................................................... --
